rinex/src/navigation/health/qzss.rs

```diff
@@ -47,48 +44,77 @@ impl QZSSHealth {
 }
 
 bitflags! {
-    /// QZSS Legacy Sv Health indication.
+    /// QZSS LNAV SV Health indication.
     /// See [Bibliography::RINEX3] and [Bibliography::QzssPnt] 5.4.1
     /// for more information.
     #[derive(Default, Debug, Clone, PartialEq, PartialOrd)]
     #[cfg_attr(feature = "serde", derive(Serialize))]
-    pub struct LegacyHealth: u64 {
-        const L1CA_B= 0x01 << 22;
-        const L1C_A = 0x01 << 21;
-        const L2    = 0x01 << 20;
-        const L5    = 0x01 << 19;
-        const L1C   = 0x01 << 18;
-        const L1C_B = 0x01 << 17;
-
+    pub struct LNAVHealth: u64 {
+        /// L1 Health
+        /// `false` = Healthy
+        /// `true` = Unhealthy
+        const L1    = 0x20;
+        /// L1 C/A Health
+        /// `false` = Healthy
+        /// `true` = Unhealthy
+        const L1_CA = 0x10;
+        /// L2 Health
+        /// `false` = Healthy
+        /// `true` = Unhealthy
+        const L2    = 0x08;
+        /// L5 Health
+        /// `false` = Healthy
+        /// `true` = Unhealthy
+        const L5    = 0x04;
+        /// L1C Health
+        /// `false` = Healthy
+        /// `true` = Unhealthy
+        const L1C   = 0x02;
+        /// L1 C/B Health
+        /// `false` = Healthy
+        /// `true` = Unhealthy
+        const L1_CB = 0x01;
     }
 }
 
-impl std::fmt::UpperExp for LegacyHealth {
+impl std::fmt::UpperExp for LNAVHealth {
     fn fmt(&self, f: &mut std::fmt::Formatter) -> std::fmt::Result {
         write!(f, "{:e}", self.bits() as f32)
     }
 }
 
 bitflags! {
-    /// QZSS CNAV Health indications.
-    /// See [Bibliography::RINEX4] and [Bibliography::QzssPnt] 5.4.1
+    /// QZSS CNAV Health indication.
+    /// See [Bibliography::RINEX4] and [Bibliography::QzssPnt] 5.4.1 for more information.
     /// for more information.
     #[derive(Default, Debug, Clone, PartialEq, PartialOrd)]
     #[cfg_attr(feature = "serde", derive(Serialize))]
-    pub struct CivilianHealth: u64 {
-        const L5 = 0x01 << 54;
-        const L2 = 0x01 << 53;
-        const L1 = 0x01 << 52;
+    pub struct CNAVHealth: u8 {
```

Comment:
Changed the bit definitions as the RINEX spec is trying to say that this value is 3 bits long, taken from bit positions 54-52 of the navigation data Message 10